concordium-consensus/src/Concordium/KonsensusV1/Types.hs

```diff
@@ -287,12 +291,14 @@ instance Serialize FinalizationEntry where
         put feFinalizedQuorumCertificate
         let QuorumCertificate{..} = feSuccessorQuorumCertificate
         put qcEpoch
```

Comment:
As pointed out in a comment in slack, the line `put qcEpoch` is now redundant, since both QCs should have the same epoch for it to be a valid `FinalizationEntry` (if that is the final decision on finalization entries).

Reply:
Addressed this, please have a look and confirm that things are as expected.